Makefile

```diff
@@ -1,3 +1,4 @@
+EPOCH_VERSION =? 0.16.0
```

Comment:
Why defaulting?

Reply:
I mean - silent unexpected default seems worse than error.

Reply:
Rethinking this, it seems wrong to me to have one version for all environments, thus I believe that this constant should be hardcoded in the terraform main file for each module e.g. https://github.com/aeternity/infrastructure/pull/64/files#diff-b89d1f9fe4d022751a376ef53e46021dR42

As TF configuration should represent the current state of the infrastructure, also we must have the option to set different epoch version per environment.